### PR TITLE
Allow to specify image for autodeploy

### PR DESCRIPTION
**Pull request by fleupold**
Dependent on and related to https://github.com/cowprotocol/k8s-autodeploy/pull/2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -4,14 +4,24 @@ const { AutoDeployApi } = require("./autodeploy");
...
 
     core.info(`Re-deploying pods ${pods} for tag ${tag}`);
     const api = new AutoDeployApi(url, token);
-    status = await api.redeploy(pods, tag);
+
+    if (pods) {
+      status = await api.redeploy("services", tag);
+    }
+    if (images) {
+      status = await api.redeploy("images", tag);
```
Comment (MartinquaXD) [inline index.js]: The new signature is `AutoDeploy.redeploy(type, targets, pod)` which seems to be used here incorrectly now.

**Review by MartinquaXD — APPROVED** [review]

File: action.yml [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
-description: 'Automatically trigger a re-deployment of Kubernetes pods.'
+name: "Auto-Deploy"
+description: "Automatically trigger a re-deployment of Kubernetes pods."
 inputs:
   pods:
-    description: 'The pods to re-deploy'
-    required: true
+    description: "The pods to re-deploy"
+  images:
+    description: "Re-deploy all pods running this image"
```
Comment (nlordell) [inline action.yml:7]: Late to the party, but should shit have:

```yaml
    required: false
```

Comment (fleupold) [reply action.yml:7]: I though default is false (https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_idrequired)

Comment (nlordell) [reply action.yml:7]: Then nothing to do. I saw `required: false` in the other entries

**Pull request merged**